Bio/SeqIO/PdbIO.py

```diff
@@ -346,6 +352,10 @@ def CifSeqresIterator(handle):
     metadata = collections.defaultdict(list)
     records = MMCIF2Dict(handle)
 
+    # raise exception if file is empty
+    if len(records) == 0:
+        raise ValueError("Empty file.")
```

Comment:
Is this the only way to get an empty dictionary back? Should the exception happen inside MMCIF2Dict instead?

Reply:
I'm now using the buffer logic used in `CifAtomIterator()` to check for emptiness.

```
buffer = StringIO()
shutil.copyfileobj(handle, buffer)

# check if file is empty
if len(buffer.getvalue()) == 0:
        raise ValueError("Empty file.")
```

Reply:
That does not look elegant just to catch an empty handle. There was good reason for CIF to use a buffer as internally it calls two parsers.

Reply:
Using `UndoHandle(handle).peekline()` instead.